### DISCUSSION TITLE
Multiselect list in start node (workflow)

### DISCUSSION THREAD

**Discussion by AndyMik90** [langgenius/dify#3457] [Suggestion]
Some popular workflows like creating social media posts. Having a multi-select input field in the start node would be great no get a better structure of nodes.
Then, we could use the already built IF/ELSE and say that if that variable contains "X" or "Y," we can direct it to the next node like with a normal select variable. 
![CleanShot 2024-04-13 at 22 50 55](https://github.com/langgenius/dify/assets/119136210/580efd4e-3598-42f6-a9c8-da463ae36c1c)

For instance I would use it to create social media posts for each of the platforms that the user selects. So that we can get the workflow for just one social media platform, but also for more if the user selects they want to crate posts for many different platforms.

**Comment by byrocuy** [comment]
Any update for this?
